app/views/my/_password_form_fields.html.erb

```diff
@@ -0,0 +1,52 @@
+<div class="form--field">
+  <label class="form--label" for="username">
+    <%= User.human_attribute_name(:login) %>
+  </label>
+  <div class="form--field-container">
+    <%= @username %>
+  </div>
+  <%= hidden_field_tag 'username', @username %>
+</div>
+
+
+
+<div class="form--field">
+  <%= styled_label_tag 'password', User.human_attribute_name(:password) %>
+  <div class="form--field-container">
+    <%= styled_password_field_tag 'password', nil, :size => 25 %>
+  </div>
+</div>
+
+
+
+<div class="form--field">
+  <%= styled_label_tag 'new_password', User.human_attribute_name(:new_password) %>
+  <div class="form--field-container">
+    <%= styled_password_field_tag 'new_password', nil, :size => 25,
+          data: { 'tooltip-target' => '#new_password_tooltip' },
+          class: 'advanced-tooltip-trigger'%>
+  </div>
+  <div class="advanced-tooltip-content" id="new_password_tooltip">
+    <%= render_password_complexity_tooltip %>
+  </div>
+</div>
+
+
+
+<div class="form--field">
+  <%= styled_label_tag 'new_password_confirmation',
+        User.human_attribute_name(:password_confirmation) %>
+  <div class="form--field-container">
+    <%= styled_password_field_tag 'new_password_confirmation', nil,
+          data: { 'tooltip-target' => '#confirm_password_tooltip' },
+          :size => 25, class: 'advanced-tooltip-trigger' %>
+  </div>
+  <div class="advanced-tooltip-content" id="confirm_password_tooltip">
+    <%= I18n.t(:confirm,
+         scope: [:activerecord, :errors, :models, :user, :attributes, :password, :match])%>
+         <br>
+     <%= I18n.t(:description,
+         scope: [:activerecord, :errors, :models, :user, :attributes, :password, :match])%>
+  </div>
```

Comment:
For accessibility reasons, we should additionally use `aria-describedby` on the form elements.